todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.kt

```diff
@@ -129,13 +129,15 @@ class TasksFragment : Fragment() {
 
     private fun setupRefreshLayout() {
         viewDataBinding.refreshLayout.run {
-            setColorSchemeColors(
-                    ContextCompat.getColor(activity, R.color.colorPrimary),
-                    ContextCompat.getColor(activity, R.color.colorAccent),
-                    ContextCompat.getColor(activity, R.color.colorPrimaryDark)
-            )
-            // Set the scrolling view in the custom SwipeRefreshLayout.
-            scrollUpChild = viewDataBinding.tasksList
+            activity?.run {
```

Comment:
Hmm I actually prefer `let` here (and on the outside of viewDataBinding.refreshLayout) to help differentiate between activity and refreshLayout.